docs/_core_features/image-generation.md

```diff
@@ -119,7 +119,7 @@ image_portrait = RubyLLM.paint(
 )
 ```
 
-> Not all models support size customization. If a size is specified for a model that doesn't support it (like Google Imagen), RubyLLM may log a debug message indicating the size parameter is ignored. Check the provider's documentation or the [Available Models Guide]({% link _reference/available-models.md %}) for supported sizes.
```

Comment:
I’ve removed the log because we’d now support passing arbitrary params. It’d be up to the user to make sure the model supports what they’re passing in.

It’s a bit of a sharp knife, but given Replicate opens the door to tons of models with tons of input signatures, allowing arbitrary params is the only way I can think of to implement. Open to other ideas though.

I’m wrapping up for today but will add this to the docs later.